### PR TITLE
Dev helm

### PR DESCRIPTION
**Pull request by sergeyberezansky**
The only missing part is publishing archives to helm repo. But local install works

### CODE REVIEW

File: deploy/helm/csi-wekafsplugin/values.yaml [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# Default values for csi-wekafs-driver.
+
+replicaCount: 1
+instanceCount: 1
```
Comment (tigrawap) [inline deploy/helm/csi-wekafsplugin/values.yaml:4]: why in values if it's not something that supposed to change?

File: deploy/kubernetes-1.18/wekafs/csi-wekafs-plugin.yaml [hunk 2]
```diff
@@ -65,106 +66,121 @@ rules:
...
-          - mountPath: /csi
-            name: socket-dir
-          - mountPath: /registration
-            name: registration-dir
-          - mountPath: /var/lib/csi-wekafs-data
-            name: csi-data-dir
-
         - name: wekafs
-          image: quay.io/weka.io/csi-wekafs:latest
+          image: quay.io/weka.io/csi-wekafs:v0.6.2-0
```
Comment (tigrawap) [inline deploy/kubernetes-1.18/wekafs/csi-wekafs-plugin.yaml:175]: why hard coded? It should come out of helm version/values

File: deploy/kubernetes-1.18/wekafs/csi-wekafs-plugin.yaml [hunk 3]
```diff
@@ -281,3 +301,134 @@ spec:
...
+          volumeMounts:
+          - mountPath: /csi
+            name: socket-dir
+          - mountPath: /registration
+            name: registration-dir
+          - mountPath: /var/lib/csi-wekafs-data
+            name: csi-data-dir
+
+        - name: wekafs
+          image: quay.io/weka.io/csi-wekafs:v0.6.2-0
```
Comment (tigrawap) [inline deploy/kubernetes-1.18/wekafs/csi-wekafs-plugin.yaml:350]: same here, hardcoded

File: pkg/wekafs/wekafs.go [hunk 4]
```diff
@@ -35,13 +35,15 @@ type wekaFsDriver struct {
 	cs             *controllerServer
 	debugPath      string
 	dynamicVolPath string
+
+	csiMode string
```
Comment (tigrawap) [inline pkg/wekafs/wekafs.go:39]: use type aliasing + consts, not arbitrary string

File: deploy/helm/csi-wekafsplugin/values.yaml (hunk 1 above)
Comment (sergeyberezansky) [reply deploy/helm/csi-wekafsplugin/values.yaml:4]: Maybe they want to change it for redundancy?

File: deploy/kubernetes-1.18/wekafs/csi-wekafs-plugin.yaml (hunk 2 above)
Comment (sergeyberezansky) [reply deploy/kubernetes-1.18/wekafs/csi-wekafs-plugin.yaml:175]: The csi-wekafs-plugin.yaml is used with legacy deploy.sh installation, not through helm.

File: deploy/kubernetes-1.18/wekafs/csi-wekafs-plugin.yaml (hunk 3 above)
Comment (sergeyberezansky) [reply deploy/kubernetes-1.18/wekafs/csi-wekafs-plugin.yaml:350]: same as above

**Review by tigrawap — APPROVED** [review]

**Pull request merged**